encoder/generate_test.go

```diff
@@ -40,8 +41,8 @@ func TestGenerate(t *testing.T) {
 
 	for _, c := range cases {
 		t.Run(c.Name, func(t *testing.T) {
-			var r cue.Runtime
-			i, err := r.Compile(c.Name+".cue", c.CUE)
+			ctx := cuecontext.New()
```

Comment:
hmm... no change necessary for this PR, but i wonder if public cuetsy functions should start taking a context argument and using it to decide things like how far we dereference a structure

Reply:
sounds nice! :) just saw that we can pass the scope parameter, probably able to do it that way.

Reply:
created a new ticket https://github.com/grafana/cuetsy/issues/29